com.unity.render-pipelines.high-definition/Documentation~/Local-Volumetric-Fog.md

```diff
@@ -41,6 +41,10 @@ The **Max Local Volumetric Fog Size** controls the maximum resolution you can us
 
 The **Max Local Volumetric Fog On Screen** controls how many Local Volumetric Fog components can appear on-screen at once. This setting also has an impact on memory.
 
+## Builtin Density Mask Texture
+
+There is a set of 3D textures that contains different noises and shapes that you can import from the HDRP samples. To do so, go to the package manager window and find the **High Definition RP** package, expand the **Samples** foldout and click on the Import button at the right of the "Local Volumetric Fog Mask Textures" sample.
```

Comment:
Suggestion, please double-check the instructions are correct :) 

HDRP includes 3D Density Mask Textures with different noise values and shapes that you can use in your scene. To use these Textures, import them from the HDRP samples package:

- Open the Package Manager window (menu: **Window** > **Package Manager**) 
- Find the **High Definition RP** package
- Expand **Samples** 
- Find the "Local Volumetric Fog Mask Textures" sample and click on the Import button to the right.

Reply:
Yes, it's good. I made the modifications :)